presto-client/src/main/java/com/facebook/presto/client/QueryResults.java

```diff
@@ -329,6 +329,7 @@ private static Object fixValue(TypeSignature signature, Object value)
             case IPADDRESS:
             case DECIMAL:
             case CHAR:
+            case "Geometry":
```

Comment:
Could there be case-sensitivity issues?

Reply:
Given that all over constants are case-sensitive as well, any issues of this kind would affect all these other types as well. I don't think this new constant is any different. Is there something I'm not thinking about?

Reply:
Based on our offline conversation, this should be ok. The only thing I'd suggest is adding the type to the enum, since it's (for now), as standard as every other type.